packages/data-model/docs/schema.dbml

```diff
@@ -42,18 +42,21 @@ table submissions {
 
 table submitted_data {
   id serial [pk, not null, increment]
+  comment varchar
```

Comment:
is this the same as "reason"?

Reply:
Yes. `comment` column stores the "reason" for a deletion

Reply:
changed to column name `deleteReason`